lib/logging/fluentd/Dockerfile

```diff
@@ -6,6 +6,8 @@ LABEL Maintainer="CanDIG Project"
 
 USER root
 
+RUN gem uninstall elasticsearch && gem install elasticsearch -v 7.17.0
```

Comment:
Why does elastic version need to be pinned?

Reply:
A new release (v8.0.0) of elasticsearch about ~3 weeks ago is incompatible with the fluentd-plugin and as such we need to pin the version for now until the release gets fixed. This is simply the solution to the weird logging errors I was mentioning in the candig2-dev channel.